### PR TITLE
Added missing parameters for SetStorageSync, and GetStorageOptions

### PR DESCRIPTION
**Pull request by sprhawk**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [ x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/uni-app/index.d.ts [hunk 1]
```diff
@@ -209,7 +209,7 @@ declare class Uni {
      *
      * 参考: [http://uniapp.dcloud.io/api/storage/storage?id=setstoragesync](http://uniapp.dcloud.io/api/storage/storage?id=setstoragesync)
      */
-    setStorageSync(key: string): any;
+    setStorageSync(key: string, value: string): any;
```
Comment (zhetengbiji) [inline types/uni-app/index.d.ts]: value: any

**Comment by zhetengbiji** [comment]
感谢您的 pr，不过 value 类型应为 any

**Review by zhetengbiji — CHANGES_REQUESTED** [review]
感谢您的 pr，不过 value 类型应为 any

File: types/uni-app/index.d.ts (hunk 1 above)
Comment (sprhawk) [reply types/uni-app/index.d.ts]: @zhetengbiji 已修改

**Comment by sprhawk** [comment]
@zhetengbiji 你好，有另一个关于types 的问题，因为这个 repo 没有 issues，所以我提问在这里：https://ask.dcloud.net.cn/question/70715
不知道这个应该如何修复？

**Pull request merged**